### PR TITLE
Removed as many references to TAO as possible from the ASR deployment notebooks

### PR DESCRIPTION
**Pull request by svenchilton**
Updated the instructions and explanatory comments in the ASR deployment notebooks. Renamed the ASR basics and ASR deployment notebooks. Any broken links are for NeMo-based fine-tuning tutorials which don't yet exist, or for notebooks whose new names haven't been merged into the main branch of the repo.

Signed-off-by: Sven Chilton <sven.chilton@gmail.com>

### CODE REVIEW

File: asr-deployment-am-and-ngram-lm.ipynb [hunk 1]
```diff
@@ -511,7 +449,26 @@
...
-    "# ! docker stop $(docker ps -a -q)"
+    "audio_file = \"<add path to .wav, .flac, .alaw, or .mulaw file>\"\n",
+    "run_inference(audio_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can stop the Riva Servicemaker container (and thus shut down Riva Server) before shutting down the Jupyter kernel."
```
Comment (LynseyFabel) [inline asr-deployment-am-and-ngram-lm.ipynb:460]:
```suggestion
    "You can stop the Riva ServiceMaker container (and thus shut down the Riva server) before shutting down the Jupyter kernel."
```

File: asr-deployment-citrinet.ipynb [hunk 2]
```diff
@@ -23,22 +23,24 @@
     "- Text-to-Speech synthesis (TTS).\n",
     "- A collection of natural language processing (NLP) services, such as named entity recognition (NER), punctuation, and intent classification.\n",
     "\n",
-    "In this tutorial, we will deploy custom acoustic model (Citrinet) trained with TAO Toolkit on Riva. <br> \n",
-    "To understand the basics of Riva ASR APIs, refer to [Getting started with Riva ASR in Python](https://github.com/nvidia-riva/tutorials/blob/stable/asr-python-basics.ipynb). <br>\n",
+    "In this tutorial, we will deploy custom acoustic model (Citrinet) trained with NeMo on Riva. <br> \n",
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:26]:
```suggestion
    "In this tutorial, we will deploy a custom acoustic model (Citrinet) trained with NeMo on Riva. <br> \n",
```

File: asr-deployment-citrinet.ipynb [hunk 3]
```diff
@@ -23,22 +23,24 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Train, Adapt, and Optimize TAO Toolkit\n",
-    "[Train Adapt Optimize (TAO) Toolkit](https://developer.nvidia.com/tao-toolkit) provides the capability to export your model in a format that can deployed using [NVIDIA Riva](https://developer.nvidia.com/riva), a highly performant application framework for multi-modal conversational AI services using GPUs. \n",
+    "## NeMo (Neural Modules) and `nemo2riva`\n",
+    "[NVIDIA NeMo](https://developer.nvidia.com/nvidia-nemo) is an open-source framework for building, training, and fine-tuning GPU-accelerated speech AI and natural language understanding (NLU) models with a simple Python interface. For practice in fine-tuning a Citrinet acoustic model with NeMo, refer to the [Citrinet fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb).\n",
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:37]:
```suggestion
    "[NVIDIA NeMo](https://developer.nvidia.com/nvidia-nemo) is an open-source framework for building, training, and fine-tuning GPU-accelerated speech AI and natural language understanding (NLU) models with a simple Python interface. To fine-tune a Citrinet acoustic model with NeMo, refer to the [Citrinet fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb).\n",
```

File: asr-deployment-citrinet.ipynb [hunk 4]
```diff
@@ -23,22 +23,24 @@
...
-    "In this notebook, you will learn how to:  \n",
-    "- Use Riva ServiceMaker to take a TAO exported `.riva` file and convert it to `.rmir`.\n",
+    "The [`nemo2riva`]() command-line tool provides the capability to export your `.nemo` model in a format that can deployed using [NVIDIA Riva](https://developer.nvidia.com/riva), a highly performant application framework for multi-modal conversational AI services using GPUs. A Python `.whl` file for `nemo2riva` is included in the [Riva Skills Quick Start](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/riva/resources/riva_quickstart) resource folder. \\[Editor's Note: This next sentence is not yet true, but hopefully will be by the time this tutorial is released.\\] You can also install `nemo2riva` with `pip`, as in the [Citrinet fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb). \n",
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:39]:
```suggestion
    "The [`nemo2riva`]() command-line tool provides the capability to export your `.nemo` model in a format that can be deployed using [NVIDIA Riva](https://developer.nvidia.com/riva), a highly performant application framework for multi-modal conversational AI services using GPUs. A Python `.whl` file for `nemo2riva` is included in the [Riva Quick Start](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/riva/resources/riva_quickstart) resource folder. \\[Editor's Note: This next sentence is not yet true, but hopefully will be by the time this tutorial is released.\\] You can also install `nemo2riva` with `pip`, as shown in the [Citrinet fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb). \n",
```

File: asr-deployment-citrinet.ipynb [hunk 5]
```diff
@@ -23,22 +23,24 @@
...
+    "\n",
+    "This tutorial explores taking a `.riva` model &mdash; the result of invoking the `nemo2riva` CLI tool (refer to the [Citrinet fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb)) &mdash; and leveraging the Riva ServiceMaker framework to aggregate all the necessary artifacts for Riva deployment to a target environment. Once the model is deployed in Riva, you can issue inference requests to the server. We will demonstrate how quick and straightforward this whole process is.\n",
+    "In this notebook, you will learn how to:\n",
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:42]:
```suggestion
    "In this tutorial, you will learn how to:\n",
```

File: asr-deployment-citrinet.ipynb [hunk 6]
```diff
@@ -136,8 +135,9 @@
...
-    "2. Refer to the [Riva ASR Pipeline Configuration](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html) documentation page if you wish to build an ASR pipeline for a supported language other than US English. To obtain the proper `riva-build` parameters for your particular application, select the acoustic model, language, and pipeline type (offline for the purposes of this tutorial) from the interactive web menu at the bottom of the first section of the page."
+    "1. If you encrypted your acoustic model and/or language model by adding the `--key` flag when invoking `nemo2riva`, or you downloaded a pre-trained model from NGC dated before 2023, you'll need to append a colon and then the key's value to the model's name in the `riva-build` command below. You might find it convenient to set a string variable named `KEY` and pass it into the appropriate `riva-build` arguments as `$KEY`. The standard encryption key for the older pre-trained models is `tlt_encode`.\n",
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:138]:
```suggestion
    "1. If you encrypted your acoustic model and/or language model by adding the `--key` flag when invoking `nemo2riva`, or you downloaded a pre-trained model from NGC dated before 2023, you'll need to append a colon and then the key's value to the model's name in the `riva-build` command, as shown below. You might find it convenient to set a string variable named `KEY` and pass it into the appropriate `riva-build` arguments as `$KEY`. The standard encryption key for the older pre-trained models is `tlt_encode`.\n",
```

File: asr-deployment-citrinet.ipynb [hunk 7]
```diff
@@ -136,8 +135,9 @@
...
+    "2. If you obtained your `.riva`-formatted acoustic model file from `tao <task> export`, you may need to add the parameter `--nn.use_trt_fp32` when invoking `riva-build`. \n",
+    "3. Refer to the [Riva ASR Pipeline Configuration](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html) documentation page if you wish to build an ASR pipeline for a supported language other than US English. To obtain the proper `riva-build` parameters for your particular application, select the acoustic model, language, and pipeline type (offline for the purposes of this tutorial) from the interactive web menu at the bottom of the first section of the page."
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:140]:
```suggestion
    "3. Refer to the [Riva ASR Pipeline Configuration documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html) if you want to build an ASR pipeline for a supported language other than US English. To obtain the proper `riva-build` parameters for your particular application, select the acoustic model, language, and pipeline type (offline for the purposes of this tutorial) from the interactive web menu at the bottom of the first section of the page."
```

File: asr-deployment-citrinet.ipynb [hunk 8]
```diff
@@ -170,9 +170,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Riva-deploy\n",
+    "### Riva-Deploy\n",
     "\n",
-    "The deployment tool takes as input one or more RMIR files and a target model repository directory. It creates an ensemble configuration specifying the pipeline for the execution and finally writes all those assets to the output model repository directory."
+    "The deployment tool takes as input one or more RMIR files and a target model repository directory. It creates an ensemble configuration specifying the pipeline for the execution and finally writes all those assets to the output model repository directory.\n",
+    "\n",
+    "**Note:** If you added an encryption key to your `.rmir` file when building it with `riva-build`, make sure to append a colon and then the key's value to the model's name in the `riva-deploy` command below."
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:177]:
```suggestion
    "**Note:** If you added an encryption key to your `.rmir` file when building it with `riva-build`, make sure to append a colon and then the key's value to the model's name in the `riva-deploy` command, as shown below."
```

File: asr-deployment-citrinet.ipynb [hunk 9]
```diff
@@ -366,7 +406,26 @@
...
-    "! docker stop $(docker ps -a -q)"
+    "audio_file = \"<add path to .wav, .flac, .alaw, or .mulaw file>\"\n",
+    "run_inference(audio_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can stop the Riva Servicemaker container (and thus shut down Riva Server) before shutting down the Jupyter kernel."
```
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:417]:
```suggestion
    "You can stop the Riva ServiceMaker container (and thus shut down the Riva server) before shutting down the Jupyter kernel."
```

File: asr-deployment-conformer-ctc.ipynb [hunk 10]
```diff
@@ -23,22 +23,24 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Train, Adapt, and Optimize TAO Toolkit\n",
-    "[Train Adapt Optimize (TAO) Toolkit](https://developer.nvidia.com/tao-toolkit) provides the capability to export your model in a format that can be deployed using [NVIDIA Riva](https://developer.nvidia.com/riva), a highly performant application framework for multi-modal conversational AI services using GPUs. \n",
+    "## NeMo (Neural Modules) and `nemo2riva`\n",
+    "[NVIDIA NeMo](https://developer.nvidia.com/nvidia-nemo) is an open-source framework for building, training, and fine-tuning GPU-accelerated speech AI and natural language understanding (NLU) models with a simple Python interface. For practice in fine-tuning a Conformer-CTC acoustic model with NeMo, refer to the [Conformer-CTC fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb).\n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:37]:
```suggestion
    "[NVIDIA NeMo](https://developer.nvidia.com/nvidia-nemo) is an open-source framework for building, training, and fine-tuning GPU-accelerated speech AI and natural language understanding (NLU) models with a simple Python interface. To fine-tune a Conformer-CTC acoustic model with NeMo, refer to the [Conformer-CTC fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb).\n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 11]
```diff
@@ -23,22 +23,24 @@
...
-    "In this tutorial, you will learn how to:  \n",
-    "- Use Riva ServiceMaker to take a TAO exported `.riva` file and convert it to `.rmir`.\n",
+    "The [`nemo2riva`]() command-line tool provides the capability to export your `.nemo` model in a format that can deployed using [NVIDIA Riva](https://developer.nvidia.com/riva), a highly performant application framework for multi-modal conversational AI services using GPUs. A Python `.whl` file for `nemo2riva` is included in the [Riva Skills Quick Start](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/riva/resources/riva_quickstart) resource folder. \\[Editor's Note: This next sentence is not yet true, but hopefully will be by the time this tutorial is released.\\] You can also install `nemo2riva` with `pip`, as in the [Conformer-CTC fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb). \n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:39]:
```suggestion
    "The [`nemo2riva`]() command-line tool provides the capability to export your `.nemo` model in a format that can be deployed using [NVIDIA Riva](https://developer.nvidia.com/riva), a highly performant application framework for multi-modal conversational AI services using GPUs. A Python `.whl` file for `nemo2riva` is included in the [Riva Quick Start](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/riva/resources/riva_quickstart) resource folder. \\[Editor's Note: This next sentence is not yet true, but hopefully will be by the time this tutorial is released.\\] You can also install `nemo2riva` with `pip`, as shown in the [Conformer-CTC fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb). \n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 12]
```diff
@@ -23,22 +23,24 @@
...
+    "\n",
+    "This tutorial explores taking a `.riva` model &mdash; the result of invoking the `nemo2riva` CLI tool (refer to the [Conformer-CTC fine-tuning tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb)) &mdash; and leveraging the Riva ServiceMaker framework to aggregate all the necessary artifacts for Riva deployment to a target environment. Once the model is deployed in Riva, you can issue inference requests to the server. We will demonstrate how quick and straightforward this whole process is.\n",
+    "In this notebook, you will learn how to:\n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:42]:
```suggestion
    "In this tutorial, you will learn how to:\n",
```

File: asr-deployment-citrinet.ipynb (hunk 5 above)
Comment (LynseyFabel) [inline asr-deployment-citrinet.ipynb:42]:
```suggestion
    "In this tutorial, you will learn how to:\n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 13]
```diff
@@ -57,7 +59,12 @@
...
+    "- A `.riva` model file that you want to deploy. You can convert a `.nemo` model file to a `.riva` model file with the `nemo2riva` command. \n",
+    "    - For information on customizing a Conformer-CTC acoustic model with NeMo and exporting the resulting model with `nemo2riva`, refer to the [Conformer-CTC finetuning tutorial](). \n",
+    "    - Alternatively, you can obtain a pre-trained Conformer-CTC `.riva` model for English ASR [here](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechtotext_en_us_conformer). \n",
+    "    - For additional general information on training NeMo models, refer to the [Training](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html#training) section of the [NeMo Models](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html) page in the [NeMo Documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/index.html). \n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:65]:
```suggestion
    "    - For more information on training NeMo models, refer to the [Training](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html#training) section in the [NeMo documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/index.html). \n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 14]
```diff
@@ -57,7 +59,12 @@
    "source": [
     "Before we get started, ensure you have:\n",
     "- Access to NVIDIA NGC and are able to download the Riva Quick Start [resources](https://ngc.nvidia.com/catalog/resources/nvidia:riva:riva_quickstart).\n",
-    "- A `.riva` model file that you want to deploy. You can obtain this from `tao <task> export` (with `export_format=RIVA`) or download a pre-trained version from the [US English Conformer NGC model page](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechtotext_en_us_conformer). For more information on training and exporting a `.riva` Conformer-CTC acoustic model, refer to the [Speech Recognition with Conformer](https://docs.nvidia.com/tao/tao-toolkit/text/asr/speech_recognition_with_conformer.html) pages in the [TAO Toolkit Documentation](https://docs.nvidia.com/tao/tao-toolkit/index.html)."
+    "- A `.riva` model file that you want to deploy. You can convert a `.nemo` model file to a `.riva` model file with the `nemo2riva` command. \n",
+    "    - For information on customizing a Conformer-CTC acoustic model with NeMo and exporting the resulting model with `nemo2riva`, refer to the [Conformer-CTC finetuning tutorial](). \n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:63]:
```suggestion
    "    - For more information on customizing a Conformer-CTC acoustic model with NeMo and exporting the resulting model with `nemo2riva`, refer to the [Conformer-CTC fine-tuning tutorial](). \n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 15]
```diff
@@ -57,7 +59,12 @@
...
+    "    - Alternatively, you can obtain a pre-trained Conformer-CTC `.riva` model for English ASR [here](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechtotext_en_us_conformer). \n",
+    "    - For additional general information on training NeMo models, refer to the [Training](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html#training) section of the [NeMo Models](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html) page in the [NeMo Documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/index.html). \n",
+    "    - For more information on Conformer-CTC's architecture, refer to the [Conformer-CTC section](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#conformer-ctc) of the [NeMo ASR Models](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html) page. \n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:66]:
```suggestion
    "    - For more information on the Conformer-CTC's architecture, refer to the [Conformer-CTC](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#conformer-ctc) section. \n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 16]
```diff
@@ -57,7 +59,12 @@
...
+    "    - For more information on the configuration files necessary for training Conformer-CTC with NeMo, refer to the [Conformer-CTC](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/configs.html#conformer-ctc) section of the [NeMo ASR Model Configuration Files](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/configs.html) page."
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:67]:
```suggestion
    "    - For more information on the configuration files necessary for training Conformer-CTC with NeMo, refer to the [Conformer-CTC](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/configs.html#conformer-ctc) section."
```

File: asr-deployment-conformer-ctc.ipynb [hunk 17]
```diff
@@ -129,13 +133,16 @@
...
-    "2. Refer to the [Riva ASR Pipeline Configuration](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html) documentation page if you wish to build an ASR pipeline for a supported language other than US English. To obtain the proper `riva-build` parameters for your particular application, select the acoustic model, language, and pipeline type (offline for the purposes of this tutorial) from the interactive web menu at the bottom of the first section of the page."
+    "1. If you encrypted your acoustic model and/or language model by adding the `--key` flag when invoking `nemo2riva`, or you downloaded a pre-trained model from NGC dated before 2023, you'll need to append a colon and then the key's value to the model's name in the `riva-build` command below. You might find it convenient to set a string variable named `KEY` and pass it into the appropriate `riva-build` arguments as `$KEY`. The standard encryption key for the older pre-trained models is `tlt_encode`.\n",
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:143]:
```suggestion
    "1. If you encrypted your acoustic model and/or language model by adding the `--key` flag when invoking `nemo2riva`, or you downloaded a pre-trained model from NGC dated before 2023, you'll need to append a colon and then the key's value to the model's name in the `riva-build` command, as shown below. You might find it convenient to set a string variable named `KEY` and pass it into the appropriate `riva-build` arguments as `$KEY`. The standard encryption key for the older pre-trained models is `tlt_encode`.\n",
```

File: asr-deployment-conformer-ctc.ipynb [hunk 18]
```diff
@@ -129,13 +133,16 @@
...
+    "2. If you obtained your `.riva`-formatted acoustic model file from `tao <task> export`, you may need to replace `--nn.fp16_needs_obey_precision_pass` with `--nn.use_trt_fp32` when invoking `riva-build`. \n",
+    "3. Refer to the [Riva ASR Pipeline Configuration](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html) documentation page if you wish to build an ASR pipeline for a supported language other than US English. To obtain the proper `riva-build` parameters for your particular application, select the acoustic model, language, and pipeline type (offline for the purposes of this tutorial) from the interactive web menu at the bottom of the first section of the page."
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:145]:
```suggestion
    "3. Refer to the [Riva ASR Pipeline Configuration documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-pipeline-configuration.html) if you want to build an ASR pipeline for a supported language other than US English. To obtain the proper `riva-build` parameters for your particular application, select the acoustic model, language, and pipeline type (offline for the purposes of this tutorial) from the interactive web menu at the bottom of the first section of the page."
```

File: asr-deployment-conformer-ctc.ipynb [hunk 19]
```diff
@@ -172,7 +179,9 @@
    "source": [
     "### Riva-Deploy\n",
     "\n",
-    "The deployment tool takes as input one or more RMIR files and a target model repository directory. It creates an ensemble configuration specifying the pipeline for the execution and finally writes all those assets to the output model repository directory."
+    "The deployment tool takes as input one or more RMIR files and a target model repository directory. It creates an ensemble configuration specifying the pipeline for the execution and finally writes all those assets to the output model repository directory.\n",
+    "\n",
+    "**Note:** If you added an encryption key to your `.rmir` file when building it with `riva-build`, make sure to append a colon and then the key's value to the model's name in the `riva-deploy` command below."
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:184]:
```suggestion
    "**Note:** If you added an encryption key to your `.rmir` file when building it with `riva-build`, make sure to append a colon and then the key's value to the model's name in the `riva-deploy` command, as shown below."
```

File: asr-deployment-conformer-ctc.ipynb [hunk 20]
```diff
@@ -329,37 +353,77 @@
...
+    "audio_file = \"<add path to .wav, .flac, .alaw, or .mulaw file>\"\n",
+    "run_inference(audio_file)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can stop all Docker containers before shutting down the Jupyter kernel. **Caution: The following command will stop all running containers.**"
+    "You can stop the Riva Servicemaker container (and thus shut down Riva Server) before shutting down the Jupyter kernel."
```
Comment (LynseyFabel) [inline asr-deployment-conformer-ctc.ipynb:426]:
```suggestion
    "You can stop the Riva ServiceMaker container (and thus shut down the Riva server) before shutting down the Jupyter kernel."
```

**Review by LynseyFabel — APPROVED** [review]
I've left some comments and have approved the review. Thank you!

File: asr-deployment-am-and-ngram-lm.ipynb [hunk 21]
```diff
@@ -58,18 +60,21 @@
...
+    "    - For information on customizing a Conformer-CTC acoustic model with NeMo and exporting the resulting model with `nemo2riva`, refer to the [Conformer-CTC finetuning tutorial](). \n",
+    "    - Alternatively, you can obtain a pre-trained Conformer-CTC `.riva` model for English ASR [here](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechtotext_en_us_conformer). \n",
+    "    - For additional general information on training NeMo models, refer to the [Training](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html#training) section of the [NeMo Models](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/core/core.html) page in the [NeMo Documentation](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/index.html). \n",
+    "    - For more information on Conformer-CTC's architecture, refer to the [Conformer-CTC section](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html#conformer-ctc) of the [NeMo ASR Models](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/models.html) page. \n",
```
[the remark this answers is not in the document]
Comment (svenchilton) [reply asr-deployment-am-and-ngram-lm.ipynb:73]: I don't think one typically uses definite articles when referring to a model, except perhaps when referring to specific instance of that model. A model's architecture is a general property, though.

File: asr-deployment-am-and-ngram-lm.ipynb [hunk 22]
```diff
@@ -58,18 +60,21 @@
...
+    "    - For more information on the configuration files necessary for training Conformer-CTC with NeMo, refer to the [Conformer-CTC](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/configs.html#conformer-ctc) section of the [NeMo ASR Model Configuration Files](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/asr/configs.html) page.\n",
```
[the remark this answers is not in the document]
Comment (svenchilton) [reply asr-deployment-am-and-ngram-lm.ipynb:74]: I'm not sure I agree with this suggestion. The first thing I'd ask would be, "Conformer-CTC section of what?"

File: asr-deployment-am-and-ngram-lm.ipynb (hunk 21 above)
[the remark this answers is not in the document]
Comment (svenchilton) [reply asr-deployment-am-and-ngram-lm.ipynb:73]: Also, section of what? It's not clear from the proposed wording.

File: asr-deployment-conformer-ctc.ipynb (hunk 15 above)
Comment (svenchilton) [reply asr-deployment-conformer-ctc.ipynb:66]: Again, I don't think "the" is necessary in front of "Conformer-CTC's architecture," and again, I don't think "refer to the Conformer-CTC section" is clear. Section of what?

File: asr-deployment-conformer-ctc.ipynb (hunk 16 above)
Comment (svenchilton) [reply asr-deployment-conformer-ctc.ipynb:67]: Section of what?

**Pull request merged**